platform/util/platform-util-uuidgenerator/util-uuidgenerator-impl/src/main/java/org/codice/ddf/platform/util/uuidgenerator/impl/UuidGeneratorImpl.java

```diff
@@ -26,16 +26,16 @@
       Pattern.compile(
           "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$");
 
-  private Boolean removeHyphens = true;
+  private Boolean useHyphens = false;
 
-  public void setRemoveHyphens(Boolean removeHyphens) {
-    this.removeHyphens = removeHyphens;
+  public void setUseHyphens(Boolean useHyphens) {
```

Comment:
Recommend changing to `boolean` or doing a null check before assigning to a `boolean`